### DISCUSSION TITLE
ctellraw for bungee

### DISCUSSION THREAD

**Discussion by PanetiX** [Zrips/CMI#6824] [Feature Requests]
CMI ctellraw for bungee would be great! :)
`/cmi ctellraw all|player|bungee <T>etc</T>etc.`

so it's possible to make network wide bbroadcasts with formatted text like clickable events

**Comment by mrfloris** [comment]
Personally I would love to see <T>-etc support for /cmi broadcast, /cmi bbroadcast, and bungee-support for /cmi ctellraw. 

It might be easiest to just add a special case to ctellraw like in this suggestion, where a "bungee" param kinda turns it into a broadcast msg with hoverable stuff to all servers.
